oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/reactive/function/client/ServletOAuth2AuthorizedClientExchangeFilterFunction.java

```diff
@@ -119,16 +109,13 @@
 
 	private static final String REQUEST_CONTEXT_OPERATOR_KEY = RequestContextSubscriber.class.getName();
 
-	private Clock clock = Clock.systemUTC();
-
 	private Duration accessTokenExpiresSkew = Duration.ofMinutes(1);
 
 	private ClientRegistrationRepository clientRegistrationRepository;
 
 	private OAuth2AuthorizedClientRepository authorizedClientRepository;
```

Comment:
Are clientRegistrationRepository and authorizedClientRepository still necessary? I think that we should be able to rely on only OAuth2AuthorizedClientManager